Sources/AppStoreConnectCLI/Helpers/Array+Helpers.swift

```diff
@@ -0,0 +1,9 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import Foundation
+
+extension Array where Element == String {
+    func isSubarray(of array: [String]) -> Bool {
```

Comment:
This isn't accurate.  This removes duplicates.  A sub array is a slice or an array.  Eg, `[a, a, b, b]` is a subarray of `[a, a, a, b, b, b]` (a slice from index 1 to 4).

You should just do the set math where you call this.  It would be more obvious what you're actually doing.